### ISSUE TITLE
[Bug]: time-input - aria-label not read by screenreader

### ISSUE THREAD

**Issue by k-luise** [lyne-design-system/lyne-components#1888]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/lyne-design-system/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Which version of Lyne Components are you using

0.35.5

### Input mode

Mouse/Keyboard

### What happened?

The aria-label of the time-input does not get read by screenreaders. The placeholder (HH:MM) is read instead.

### Steps to reproduce the issue

1. Set aria-label property of time-input
2. Tab to it and listen to the screenreader

**Comment by jeripeierSBB** [comment]
@k-luise Can you add details with which screenreaders it fails? At least with VoiceOver it seems to work properly.

**Comment by jeripeierSBB** [comment]
We'd like to refactor it as following (similar to sbb-datepicker):
```
<sbb-form-field>
 <label for="id1"></label>
 <sbb-time-input></sbb-time-input>
 <input id="id1">
<sbb-form-field>
```

**Comment by k-luise** [comment]
It only fails with NVDA.
If we add the aria label in the input element in the shadow DOM it works.
